firefox-ios/Client/Frontend/Autofill/Address/Edit/EditAddressViewController.swift

```diff
@@ -120,20 +129,16 @@ class EditAddressViewController: UIViewController, WKNavigationDelegate, WKScrip
         guard let webView else { return }
         webView.evaluateJavaScript("getCurrentFormData();") { [weak self] result, error in
             if let error = error {
```

Comment:
I would move the whole block instead of only the log methods:

```swift
private func unwrapResult(result: ..., error: ...) -> [String: Any]? {
   if let error {
       // log error
       return nil
   }

   guard let result = result as? [String: Any] else {
       // log error
       return nil
   }
   return result
}
```
What do u think @ionixjunior

Reply:
I initially tried to avoid extracting the code into a new function that returns `nil`when something goes wrong. However, this approach makes sense, especially since there are other places where the logging function is used internally. So, put a new log function to using only in one place is a little bit weird. I really liked your suggestion! Additionally, I passed the `logger` as a parameter instead of using `self`. @FilippoZazzeroni, could you please check if this update works?